src/main/java/org/prebid/server/bidder/rtbhouse/RtbhouseBidder.java

```diff
@@ -101,10 +106,60 @@ private static List<BidderBid> extractBids(BidResponse bidResponse) {
                 .filter(Objects::nonNull)
                 .flatMap(Collection::stream)
                 .filter(Objects::nonNull)
-                .map(bid -> BidderBid.of(bid, BidType.banner, bidResponse.getCur()))
+                .map(bid -> resolveBidderBid(bid, bidRequest, bidResponse, bidderErrors))
                 .toList();
     }
 
+    private BidderBid resolveBidderBid(Bid bid, BidRequest bidRequest,
+                BidResponse bidResponse, List<BidderError> bidderErrors) {
+        final String currency = bidResponse.getCur();
```

Comment:
Please, format this lines like in example above.

Reply:
Done